fronts-client/src/util/card.ts

```diff
@@ -318,10 +332,30 @@ const getFeastCollectionFromFeedDrop = (
 const getArticleEntitiesFromFeedDrop = (
 	capiArticle: CapiArticle,
 	isEdition: boolean,
+	to?: PosSpec,
+	collectionType?: string,
```

Comment:
non-blocking: are we able to tighten this type to a list of collectionTypes (FLEXIBLE_GENERAL_NAME, FLEXIBLE_SPECIAL_NAME) etc rather than string?

Reply:
Hm, yeah, good point. I think that would probably mean introducing all the collection types though and ideally updating the other parts of the application where it gets used?
<img width="329" alt="Screenshot 2025-03-05 at 15 28 57" src="https://github.com/user-attachments/assets/6fc3bf23-e3a2-4be4-8b73-ca64d7432272" />

Makes me think it's best left to another PR.